Content.Server/Ninja/Components/TargetGridComponent.cs

```diff
@@ -0,0 +1,13 @@
+namespace Content.Server.Ninja.Components;
+
+/// <summary>
+/// Used by space ninja to indicate what station grid to head towards.
+/// </summary>
+[RegisterComponent]
+public sealed class TargetGridComponent : Component
+{
+    /// <summary>
+    /// The grid uid being targeted.
+    /// </summary>
+    public EntityUid Grid = EntityUid.Invalid;
```

Comment:
nullable instead of invalid

Reply:
its not optional i just thought i had to initialize it like that when i wrote it